app/models/bottleneck_event.rb

```diff
@@ -11,10 +11,11 @@ def self.last_created_on(obj)
   end
 
   def self.generate_future_events(obj)
-    _log.info("Generating future bottleneck events for: [#{obj.class} - #{obj.name}]...")
+    log_message_uniq_prefix = "Generating future bottleneck events for: [#{obj.class} - #{obj.name}]..."
+    _log.info(log_message_uniq_prefix)
```

Comment:
not sure if this `_log` is even necessary. It may result in a noop if generating is skipped.
I would actually remove this